sap-hana-mixin/dashboards/sap-hana-instance-overview.libsonnet

```diff
@@ -0,0 +1,1237 @@
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'auto',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'none',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+          {
+            color: 'red',
+            value: 80,
+          },
+        ],
+      },
+      unit: 'decmbytes',
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'bottom',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'multi',
+      sort: 'desc',
+    },
+  },
+  pluginVersion: '9.5.0-cloud.5.a016665c',
+};
+
+local topSQLQueriesByAverageTimePanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'sum by (job, sid, host, sql_hash) (hanadb_sql_top_time_consumers_mu{job=~"$job", sid=~"$sid", host=~"$host"})',
+      datasource=promDatasource,
+      legendFormat='{{host}} - hash: {{sql_hash}}',
+    ),
+  ],
+  type: 'timeseries',
+  title: 'Top SQL queries by average time',
+  description: 'Top statements by time consumed over all executions for the SAP HANA instance',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'palette-classic',
+      },
+      custom: {
+        axisCenteredZero: false,
+        axisColorMode: 'text',
+        axisLabel: '',
+        axisPlacement: 'auto',
+        barAlignment: 0,
+        drawStyle: 'line',
+        fillOpacity: 0,
+        gradientMode: 'none',
+        hideFrom: {
+          legend: false,
+          tooltip: false,
+          viz: false,
+        },
+        lineInterpolation: 'linear',
+        lineWidth: 1,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'auto',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'none',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+          {
+            color: 'red',
+            value: 80,
+          },
+        ],
+      },
+      unit: 'µs',
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'bottom',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'multi',
+      sort: 'desc',
+    },
+  },
+  pluginVersion: '9.5.0-cloud.5.a016665c',
+};
+
+local traceLogsPanel = {
+  datasource: lokiDatasource,
+  targets: [
+    {
+      datasource: lokiDatasource,
+      editorMode: 'code',
+      expr: '{filename=~"/opt/hana/shared/ID0/HDB00/hana-0/trace/.*.trc"} |=``',
```

Comment:
@kuiperda Is "ID0/HDB00/hana-0" going to change from setup to setup? if so this should be more generic

Reply:
Good catch, fixed